templates/user/profile.html

```diff
@@ -19,34 +19,44 @@
 <header>
   <div class="profile-intro wrapper">
     <figure>
-      <img name=avatar src={{ user.avatar }} width=100 height=100 onerror="this.src='{{ url_for('static', filename='images/avatar.png') }}'" alt="">
+      <img class=avatar-img
+        name=avatar
+        src={{ user.avatar }}
+        width=100
+        height=100
+        onerror="this.src='{{ url_for('static', filename='images/avatar.png') }}'">
     </figure>
     <div>
       <h1 name=username>{{ user.username or 'Insert a username' }}</h1>
       <p name=about>{{ user.about|htmlnewline or 'Introduce yourself here' }}</p>
+      {% if user.allow_community %}
+      <a href={{ url_for_trips(user) }} data-latlng={{ user.current_location|join(',') }}></a>
+      {% else %}
+      <p class=private>
```

Comment:
`<a class=private href={{ url_for('privacy') }}>…`  instead?

Reply:
yeah it's a good idea !